addons/mail/static/src/components/composer/composer.xml

```diff
@@ -142,7 +142,7 @@
             <t t-if="props.hasSendButton">
                 <button class="o_Composer_actionButton o_Composer_button o_Composer_buttonSend btn btn-primary"
                     t-att-class="{
-                        'o-last': messaging.device.isMobile or !props.hasDiscardButton,
+                        'o-last': messaging.device.isMobile or !composerView.hasDiscardButton,
```

Comment:
Not sure what `o-last` is and how it relates to not having the discard button. Perhaps this could be clarified by introducing a new field?

Reply:
This needs further code changes, namely have `ComposerView` full awareness of available actions, so that the "last" one can become just a relational field.